requirements.md

```diff
@@ -1,14 +1,14 @@
 ## Requirements
 
-* **decentralized/self-sovereign** (encryption, signing, authn, and authz should all be based on control of DIDs rathern than on CAs, IDPs, etc)
+* **decentralized/self-sovereign** (encryption, signing, authn, and authz should all be based on control of DIDs rather than on CAs, IDPs, etc)
 
 * **transport independent** (HTTPS, bluetooth, AMQP, SMTP, NFC, and so forth)
 
 * **routable** (like email, A can talk to B without a direct connection to B, by requesting that a message be forwarded--needed to mix and match transports; also, could pass through a mix network or other infrastructure that isn’t aware of it as anything other than blobs to be moved)
 
 * **secure** (tamper-proof, MITM impossible, composed of primitives widely considered best practice, vetted by experts, has formal security proofs, security guarantees of a given message are known)
 
-* **private** (third parties can’t learn who’s communicating about what, when--broader than “confidentiality” in CIA because metadata like timing and identities of parties is also protected; also, possible for a sender to be anonymous to recipient) 
+* **private** (third parties can’t learn who’s communicating about what, when--broader than “confidentiality” in CIA {huh what does this mean?} because metadata like timing and identities of parties is also protected; also, possible for a sender to be anonymous to recipient) 
```

Comment:
CIA is a common term used in security to indicate "Confidentiality, Integrity, Authenticity" the "A" can sometimes mean "Availability" as well.

Here's some good references: https://developer.mozilla.org/en-US/docs/Archive/Security/Confidentiality,_Integrity,_and_Availability

https://security.stackexchange.com/questions/148173/authenticity-confidentiality-integrity-general-questions

Reply:
Yes, I understand.   CIA, and many other TLA's have multiple or common meanings.  When using an acronym in a technical document, its best practice to define it upon first use.   I'll make that edit.

Reply:
Make sense, I saw your PR that added it and approved. Good work!